packages/editor/src/ide.ts

```diff
@@ -42,10 +44,31 @@ export class JsonEditorIde extends HTMLElement implements Editor {
     return getSchema(this._store.getState());
   }
 
+  /**
+   * Filtering selected list item from available items
```

Comment:
While we're at it, let's update the JSDoc here as well: Rephrase to "Set the predicate function that is used by the array renderer to filter container properties'